Tests/MoyaProviderIntegrationTests.swift

```diff
@@ -299,6 +299,49 @@ final class MoyaProviderIntegrationTests: QuickSpec {
                 }
             }
         }
+
+        // Resolves ValidationType not working with multipart upload #1590
+        describe("a provider performing a multipart upload with Alamofire validation") {
+            let provider = MoyaProvider<HTTPBin>()
+            let formData = HTTPBin.createTestMultipartFormData()
+            it("returns an error for status code different than 287") {
```

Comment:
last one, can we add few empty lines to the code? looks really stacked 😅 

e.g. before `it("")`, in `it("")` we could use a 3 blocks scenario similar to Act-Act-Assert which would add new line before `waitUntil` and one before expects.